.yamato/upm-ci-universal.yml

```diff
@@ -105,7 +104,7 @@ android_apis:
     - pip install unity-downloader-cli --extra-index-url https://artifactory.internal.unity3d.com/api/pypi/common-python/simple --upgrade
 
     - cd TestProjects/{{ project.folder }} && unity-downloader-cli --source-file ../../unity_revision.txt {% for component in platform.components %}-c {{ component }} {% endfor %}--wait --published-only
-
+    - cd Tools && powershell -command ". .\Unity.ps1; Set-ScreenResolution -width 1920 -Height 1080"
```

Comment:
this line is already in master, so I'm confused as to why this is showing as changed in this PR

Reply:
Ehn I'm snot sure why this review request was added it must be some sort of tooling? And also not sure why it shows up. Master was merged yesterday > 182d5cc